recipe/meta.yaml

```diff
@@ -1,8 +1,29 @@
-{% set version = "0.5.11" %}
 {% set org = "pyne" %}
+{% set version = "0.5.11" %}
+{% set build = "0" %}
 {% set sha256 = "ba05917b0de446ec520857655e77967a783e3bc723299faf5989f532d6b0647a" %}
 {% set variant = "openblas" %}
 
+# ensure moab is defined (needed for conda-smithy recipe-lint)
+{% set moab = moab or 'nomoab' %}
+# ensure dagmc is defined (needed for conda-smithy recipe-lint)
+{% set dagmc = dagmc or 'nodagmc' %}
+
+
+{% if moab != 'nomoab' %}
+{% set prefix = "moab_" + mpi %}
```

Comment:
is it ok to use double quotes? I can comb through for consistency in my follow-up  PR (so don't wory about it) -- just checking

Reply:
that's what they did in the Moab recipe :s don't know if they are important....
probably not :)